arbiter-core/src/environment.rs

```diff
@@ -129,7 +129,7 @@ pub struct Environment {
 
     /// The [`EVM`] that is used as an execution environment and database for
     /// calls and transactions.
-    pub(crate) evm: EVM<CacheDB<EmptyDB>>,
+    evm: EVM<CacheDB<EmptyDB>>,
```

Comment:
Remind me why we are doing this again.

Reply:
Because we don't actually reference the `evm` field of `Environment` anywhere else in the crate, so we are safe to keep its visibility to private. I feel it's usually good to minimize visibility to the lowest point.